.pre-commit-config.yaml

```diff
@@ -12,7 +12,7 @@ repos:
     rev: v2.7.1
     hooks:
       - id: prettier
-        exclude: tests_requre/openshift_integration/test_data/*
+        exclude: tests_requre/openshift_integration/test_data/.*
```

Comment:
I don't see any `tests_requre/`. Isn't this a copy-pasta relic from some other project?

Reply:
Right! Removed it.